### PR TITLE
docs: add report-issue button to header

### LINKED ISSUE #587: Move Report an Issue and Version History buttons into the blue header
See how Python has the Report an issue button in the top right http://googlecloudplatform.github.io/gcloud-python/latest/

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #587 
Closes #589 

screen shot 2015-05-20 at 12 44 07 pm

Preview: http://stephenplusplus.github.io/gcloud-node/#/docs/master

### CODE REVIEW

File: docs/site/css/main.css [hunk 1]
```diff
@@ -298,6 +298,32 @@ ul {
...
+    border-radius: 8px;
+    border: 0;
+    border-right: 2px solid #A0C2F4;
+    border-bottom: 2px solid #A0C2F4;
+    padding: 11px 10px;
+    font-size: .8em;
+    transition: border-color 1s, border-width .1s;
+}
+
+.page-header.fixed .v-btn:hover {
```
Comment (ryanseys) [inline docs/site/css/main.css]: Does this do anything?

Comment (ryanseys) [inline docs/site/css/main.css]: extra ping to @stephenplusplus

**Comment by stephenplusplus** [comment]
Thanks for fixing it up for me. All of us played a committing-role in this one. Good teamwork!

**Comment by ryanseys** [comment]
@stephenplusplus uh oh... I didn't fix anything :) I just merged because it was pretty low risk.

**Pull request merged**